apps/webapp/web/router.ex

```diff
@@ -0,0 +1,38 @@
+require Starwars.Router
+require Greeter.Router
+
+defmodule Webapp.Router do
+  use Webapp.Web, :router
+
+  pipeline :browser do
+    plug :accepts, ["html"]
+    plug :fetch_session
+    plug :fetch_flash
+    plug :protect_from_forgery
+    plug :put_secure_browser_headers
+  end
+
+  pipeline :api do
+    plug :accepts, ["json"]
+  end
+
+  scope "/", Webapp do
+    pipe_through :browser # Use the default browser stack
+
+    get "/", PageController, :index
+  end
+
+  # act as final catch
+  scope "/starwars" do
+    forward "/", Starwars.Router
+  end
+
+  scope "/greeter" do
+    forward "/", Greeter.Router
```

Comment:
They are not accessible unless you add their endpoint to the supervisor tree, but then you will have to deal with the ports and those things.

Now we can run:  `mix phoenix.server`
in the umbrella project and it will say: `Running Webapp.Endpoint with Cowboy using http://localhost:4000` 
So the Webapp.Endpoint is the one we are using.